releases/development/master/extra/install-chef-suse.sh

```diff
@@ -572,6 +586,11 @@ check_repo_tag key     11.3 SUSE-Cloud-5-Updates
 check_repo_tag summary 11.3 SLE11-HAE-SP3-Pool     'SUSE Linux Enterprise High Availability Extension 11 SP3' 'false'
 check_repo_tag repo    11.3 SLE11-HAE-SP3-Updates  'updates://zypp-patches.suse.de/autobuild/SLE_HAE/11-SP3/update/x86_64' 'false'
```

Comment:
Actually, @vuntz, does `$REPOS_SKIP_CHECK` still really do much? Can't I stick 'false' as the last arg to `check_repo_tag` and get the same effect?